Classes/Service/StatisticsService.php

```diff
@@ -309,4 +310,19 @@ public function getWhereJobInterval($dates, $table = 'tx_ats_domain_model_job')
 
         return $whereClause;
     }
+
+    /**
+     * Sums the values of an array on a specific key.
+     *
+     * @param      array   $statisticArray
+     * @param      string  $key
+     */
+    protected function getTotalNumber($statisticArray, $key)
+    {
+        $total = 0;
+        for ($i = 0; $i < count($statisticArray); $i++) {
```

Comment:
Instead of a loop you can combine `array_column()` with `array_sum()` to make this a oneliner.